cmd/db_check.go

```diff
@@ -34,5 +34,10 @@ func runDBCheckCmd(_ *cobra.Command, _ []string) error {
 		return stderrPrintLnf("No update available")
 	}
 
-	return stderrPrintLnf("Update available!")
+	fmt.Println("Update available!")
+	fmt.Printf("Current DB version %d was built on %s\n", currentDBMetadata.Version, currentDBMetadata.Built.String())
+	fmt.Printf("Update DB version %d was built on %s\n", updateDBEntry.Version, updateDBEntry.Built.String())
+	fmt.Printf("Update DB URL: %s\n", updateDBEntry.URL.String())
```

Comment:
I think just one more line here might be good for the user.

`You can run 'grype db update' to update to the latest db`